src/pages/news.js

```diff
@@ -53,11 +55,11 @@ const NewsPage = props => {
       </Typography>
       <Typography variant="body2">
         {t("waiting_time.last_updated")}
-        {_.get(data.allGoogleNews, "edges[0].node.date", "")}{" "}
-        {_.get(data.allGoogleNews, "edges[0].node.time", "")}
+        {withLanguage(i18n, data.allGoogleNews.edges[0].node, "gn")[0].date}{" "}
+        {withLanguage(i18n, data.allGoogleNews.edges[0].node, "gn")[0].time}
       </Typography>
-      {data.allGoogleNews.edges.map((node, index) => (
-        <div key={index} children={item(node)} />
+      {news.map((node, index) => (
+        <div key={index} children={item({node: node})} />
```

Comment:
`{node}`

Reply:
Cannot, I keep the same function for item({ node })

Reply:
i mean we should use the simple form for `{node: node}`
that is `{node}`